### PR TITLE
Add project description

### PR DESCRIPTION
**Pull request by platisd**
## Description
Adds more information in the project's main readme file to make it easier for visitors or possible collaborators to understand what `ins` is all about.

## Solved issue(s)
#76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
 # Indoor Navigation System [![Build Status](https://travis-ci.org/platisd/indoor-navigation-system.svg?branch=master)](https://travis-ci.org/platisd/indoor-navigation-system) [![codecov](https://codecov.io/gh/platisd/indoor-navigation-system/branch/dev/graph/badge.svg)](https://codecov.io/gh/platisd/indoor-navigation-system)
 
-[WIP] An indoor navigation system to guide users towards their colleagues' desks.
+[WIP] An indoor navigation system that guides you to the desk of your colleagues.
+
+## What?
+INS is a navigation system that utilizes a fleet of portable embedded devices, which have the ability to provide positioning information about themselves, to guide its user towards the desk of an employee or a colleague. The main scenario begins by searching for an employee on a mobile client. Then the system provides information regarding the whereabouts of the said employee's desk as well as navigational instructions on how to reach it.
```
Comment (samup4web) [inline README.md]: The embedded device themselves do not provide positioning information, but the rssi & AP info, right?. That information will only be determined on the backend and propagated towards the client applications.

**Review by samup4web — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (platisd) [reply README.md]: Yeah, I thought that this was made clear (i.e. they just provide positioning information and cannot position themselves) but apparently it's not. What about this instead:

>..... a fleet of portable embedded devices, **the location of which can be calculated**, to guide its user towards...

Comment (platisd) [reply README.md]: Or even simpler: "which can be located"

**Review by samup4web — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (samup4web) [reply README.md]: Yeah, that should work. Good job.

**Review by samup4web — CHANGES_REQUESTED** [review]
was approved by mistake. Now requesting change...

**Review by samup4web — APPROVED** [review]

**Pull request merged**